config/crds/verify_v1beta1_metadata.yaml

```diff
@@ -48,6 +48,8 @@ spec:
                   type: string
                 contactSurname:
                   type: string
+                encryptionCertificate:
```

Comment:
Perhaps it would be kinder to future generations to call this `samlEncryptionCertificate`?